packages/deployment-service/cdk/lib/resolve-production-OAC-custom-resource.ts

```diff
@@ -12,7 +12,7 @@ export class ResolveProductionOACCustomResource extends Construct {
         handler: 'dist/resolve-production-apply-OAC-to-all-distros.resolveProductionApplyOACToAllDistros',
         code: aws_lambda.Code.fromAsset('bundle/resolve-production-apply-OAC-to-all-distros.zip'),
         memorySize: 512,
-        timeout: Duration.minutes(5),
+        timeout: Duration.minutes(10),
```

Comment:
Probably increase this to 15 mins just to be safe then we've got the maximum time allocated